bin/spice/cmd/trace.go

```diff
@@ -76,6 +79,9 @@ $ spice trace ai_chat --id chatcmpl-At6ZmDE8iAYRPeuQLA0FLlWxGKNnM
 
 Include the input and truncate to 120 characters (default is 80).
 $ spice trace ai_chat --include-input --truncate=120
+
+# returns the SQL query for the trace
+$ spice trace ai_chat --sql
```

Comment:
@Jeadie can we make this `--output=sql`? Then in the future, we can do `--output=table` or `--output=csv`